pkg/odo/cli/storage/list.go

```diff
@@ -2,7 +2,13 @@ package storage
 
 import (
 	"fmt"
+	"github.com/openshift/odo/pkg/devfile"
+	devfileParser "github.com/openshift/odo/pkg/devfile/parser"
```

Comment:
camel cases shouldn't be used in golang.

Reply:
I mean, for package names and aliases.

Reply:
fixed